tools/dynamo/gb_id_mapping.py

```diff
@@ -0,0 +1,146 @@
+            elif "*graph_break_hints.DYNAMO_BUG" or "graph_break_hints.DYNAMO_BUG" in hint:
+                expanded_hints.extend(graph_break_hints.DYNAMO_BUG)
+            elif "*graph_break_hints.DIFFICULT" or "graph_break_hints.DIFFICULT" in hint:
+                expanded_hints.extend(graph_break_hints.DIFFICULT)
+            elif "*graph_break_hints.FUNDAMENTAL" or "graph_break_hints.FUNDAMENTAL" in hint:
+                expanded_hints.extend(graph_break_hints.FUNDAMENTAL)
+            elif "*graph_break_hints.SUPPORTABLE" or "graph_break_hints.SUPPORTABLE" in hint:
+                expanded_hints.extend(graph_break_hints.SUPPORTABLE)
+            elif "*graph_break_hints.CAUSED_BY_EARLIER_GRAPH_BREAK" or "graph_break_hints.CAUSED_BY_EARLIER_GRAPH_BREAK" in hint:
+                expanded_hints.extend(graph_break_hints.CAUSED_BY_EARLIER_GRAPH_BREAK)
+            elif "*graph_break_hints.INFERENCE_MODE" or "graph_break_hints.INFERENCE_MODE" in hint:
+                expanded_hints.extend(graph_break_hints.INFERENCE_MODE)
+            else:
+                expanded_hints.append(hint)
+        else:
+            expanded_hints.append(hint)
+
+    return expanded_hints
+
+
+def find_unimplemented_v2_calls():
+    dynamo_dir = Path(__file__).parent.parent.parent / "torch" / "_dynamo"
+    results = []
+
+    for file_path in dynamo_dir.glob("**/*.py"):
+        with open(file_path, "r") as f:
+            source = f.read()
+            try:
+                tree = ast.parse(source)
+
+                for node in ast.walk(tree):
+                    if isinstance(node, ast.Call) and hasattr(node, "func"):
+                        is_unimplemented = False
+                        if (
+                            isinstance(node.func, ast.Name)
+                            and node.func.id == "unimplemented_v2"
+                        ):
+                            is_unimplemented = True
+                        elif (
+                            isinstance(node.func, ast.Attribute)
+                            and node.func.attr == "unimplemented_v2"
+                        ):
+                            is_unimplemented = True
+
+                        if is_unimplemented:
+                            info = {
+                                "gb_type": None,
+                                "context": None,
+                                "explanation": None,
+                                "hints": [],
+                                "from_exc": None,
+                            }
+
+                            for kw in node.keywords:
+                                if kw.arg in info:
+                                    param_source = get_source_segment(source, kw.value)
+                                    if param_source:
+                                        info[kw.arg] = clean_string(param_source)
+                                    elif isinstance(kw.value, ast.Constant):
```

Comment:
When is param_source empty? When will the other branches of this if-statement run?